release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2842,8 +2842,67 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+[id="ocp-4-17-2-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, the approval mechanism for certificate signing requests (CSRs) failed because the node name and internal DNS entry, which is case insensitive, on a CSR did not match in terms of character case differences. With this release, an update to the approval mechanism for CSRs skips case-sensitive checks so that a CSR with a matching node name and internal DNS entry does not fail the check because of character case differences. (link:https://issues.redhat.com/browse/OCPBUGS-43312[*OCPBUGS-43312*])
+
+* Previously, a port conflict on the Cloud Credential Operator (CCO) and the `assisted-service` object caused cluster installations on {vmw-first} to fail. With this release, the installation program ignores the `pprof` module in the `assisted-service` object so that the port conflict no longer exists. (link:https://issues.redhat.com/browse/OCPBUGS-43069[*OCPBUGS-43069*])
+
+* Previously, when you attempted to use the `oc import-image` command to import an image in a {hcp} cluster, the command failed because of access issues with a private image registry. With this release, an update to `openshift-apiserver` pods in a {hcp} cluster resolves names that use the data plane so that the `oc import-image` command now works as expected with private image registries. (link:https://issues.redhat.com/browse/OCPBUGS-43051[*OCPBUGS-43051*])
+
+* Previously, for managed services on {hcp}, audit logs were sent to a local webhook service, `audit-webhook`. This caused issues for {hcp} pods that sent audit logs through the `konnectivity` service. With this release, `audit-webhook` is added to the list of `no_proxy` hosts so that {hcp} pods can send auti logs to the `audit-webhook` service. (link:https://issues.redhat.com/browse/OCPBUGS-42974[*OCPBUGS-42974*])
+
+* Previously, when you used the Agent-based Installer to install a cluster, `assisted-installer-controller` timed out or exited the installation process depending on whether `assisted-service` was unavailable on the rendezvous host. This situation caused the cluster installation to fail during CSR approval checks. With this release, an update to `assisted-installer-controller` ensures that the controller does not timeout or exit if `assisted-service` is unavailable. The CSR approval check now works as expected. (link:https://issues.redhat.com/browse/OCPBUGS-42839[*OCPBUGS-42839*])   
+
+* Previously, running the `openshift-install gather bootstrap --dir <workdir>` command might cause the installation program to skip analyzing analyze collected logs. The command would output the following message:
```

Comment:
```suggestion
* Previously, running the `openshift-install gather bootstrap --dir <workdir>` command might cause the installation program to skip analyzing the analyze collected logs. The command would output the following message:
```

Reply:
I stumbled over the `analyzing analyze collected logs`. Maybe: `command might cause the installation program to skip the analysis of the collected logs` or some other combination that avoids the double analysis formation.